app/src/main/java/com/android/sample/receivers/NotificationReceiver.kt

```diff
@@ -0,0 +1,29 @@
+package com.android.sample.receivers
+
+import android.app.NotificationManager
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import androidx.core.app.NotificationCompat
+import com.android.sample.R
+
+class NotificationReceiver : BroadcastReceiver() {
+  override fun onReceive(context: Context, intent: Intent) {
+    val activityId = intent.getIntExtra("activityId", 0)
+    val activityName = intent.getStringExtra("activityName") ?: "Activity"
+    val title = intent.getStringExtra("notificationTitle") ?: "Activity Reminder"
+
```

Comment:
I don't think it is a good idea to simply rename it to Activity. Maybe also discard the notification in this case.

Reply:
Changes made in new code! Will push soon